### ISSUE TITLE
Wrong zhuyin to pinyin for syllables ending with ㄨ

### ISSUE THREAD

**Issue by tsroten** [tsroten/zhon#23]
[@mthewissen reports](https://github.com/tsroten/dragonmapper/issues/16):

That is, when they have the first intonation:

```
from dragonmapper import hanzi, transcriptions
print transcriptions.zhuyin_syllable_to_pinyin(u'ㄓㄨˋ') # works
print transcriptions.zhuyin_syllable_to_pinyin(u'ㄔㄨ') # does not work
```

I traced things down to the following:

```
def _parse_zhuyin_syllable(unparsed_syllable):
    """Return the syllable and tone of a Zhuyin syllable."""
    zhuyin_tone = unparsed_syllable[-1]
    if zhuyin_tone in zhon.zhuyin.characters:
        syllable, tone = unparsed_syllable, '1'
    elif zhuyin_tone in zhon.zhuyin.marks:
        for tone_number, tone_mark in _ZHUYIN_TONES.items():
            if zhuyin_tone == tone_mark:
                syllable, tone = unparsed_syllable[:-1], tone_number
    else:
        raise ValueError("Invalid syllable: %s" % unparsed_syllable)

    return syllable, tone
```

For some reason, there is no ㄨ in `zhon.zhuyin.characters`? (also no ㄩ)

**Comment by tsroten** [comment]
Fixed in #24.

**Comment by tsroten** [comment]
This is now released on PyPi.
